schema/vrs.yaml

```diff
@@ -396,27 +387,77 @@ definitions:
     required: [ "type", "gene_id" ]
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # Range
+  # Numerics, Comparators, and Ranges
+
+  IntegerOrRange:
+    description: >-
+      A set of classes that are used to represent either single integer values
+      (such as SimpleInteger) or integer ranges (such as IntegerRange).
+    oneOf:
+      - $ref: "#/definitions/IntegerValue"
+      - $ref: "#/definitions/IntegerRange"
+    discriminator:
+      propertyName: type
 
-  Range:
+  IntegerValue:
```

Comment:
This, too, feels like an unneeded abstraction. I'm skeptical about this class as a concept.

In addition, the name is problematic because there's not obvious linguistic distinction between IntegerValue and SimpleInteger (or Integer with the previous suggestion).

This confusion is borne of the need to name an abstraction that provides little real distinguishing value.

Reply:
Okay, agreed. Will remove.